libindy/Cargo.lock

```diff
@@ -158,7 +158,7 @@ dependencies = [
  "etcommon-rlp 0.2.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "generic-array 0.8.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "hex 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "indy-crypto 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "indy-crypto 0.1.6-dev-25 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
@vimmerru should we release new stable version of IndyCrypto before merging this PR

Reply:
I think so